specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1316,11 +1324,70 @@
           "readOnly": true,
           "$ref": "#/definitions/ProvisioningState",
           "description": "The managed application provisioning state."
+        },
+        "billingDetails": {
```

Comment:
Please update the examples with examples of what all these new properties look like

Reply:
None of the new properties are required, and a lot of them are read only. I added them to both get examples. 
We are working on an example document, which will cover a number of different scenarios. The example here will be a minimum of a working request.